apps/strapi-cms/src/api/product/content-types/product/schema.json

```diff
@@ -0,0 +1,61 @@
+{
+  "kind": "collectionType",
+  "collectionName": "products",
+  "info": {
+    "singularName": "product",
+    "pluralName": "products",
+    "displayName": "Product",
+    "description": ""
+  },
+  "options": {
+    "draftAndPublish": true
+  },
+  "pluginOptions": {
+    "i18n": {
+      "localized": true
+    }
+  },
+  "attributes": {
+    "name": {
+      "pluginOptions": {
+        "i18n": {
+          "localized": true
+        }
+      },
+      "type": "string",
+      "required": true,
+      "unique": true
+    },
+    "description": {
+      "pluginOptions": {
+        "i18n": {
+          "localized": true
+        }
+      },
+      "type": "text"
+    },
+    "logo": {
+      "type": "media",
```

Comment:
Media content will be served through a CDN. We have two options to proceed: configure the plugin now ([here an example of the configuration](https://github.com/pagopa/b2b-portals/blob/cc58dd3/apps/strapi-cms/config/plugins.ts#L2-L19)) or add a task to the backlog to do it later. If we choose to add a task, we can insert the link to the task in this PR. In the task, we can also include the link to this PR